components/patient-data/pageobjects/src/main/java/org/phenotips/data/test/po/PatientRecordEditPage.java

```diff
@@ -40,20 +40,8 @@
     @FindBy(id = "PhenoTips.PatientClass_0_external_id")
     WebElement patientIdentifier;
 
-    @FindBy(id = "PhenoTips.PatientClass_0_last_name")
-    WebElement patientLastName;
-
-    @FindBy(id = "PhenoTips.PatientClass_0_first_name")
-    WebElement patientFirstName;
-
-    @FindBy(id = "xwiki-form-gender-0-0")
-    WebElement patientGenderMale;
-
-    @FindBy(id = "xwiki-form-gender-0-1")
-    WebElement genderFemale;
-
-    @FindBy(id = "HFamilyhistory")
-    WebElement familyHistorySectionTitle;
+    @FindBy(css = "#inline > div:nth-child(3) > div.clear.clinical-info.family-info.chapter.collapsed > span > span.buttonwrapper.show > button")
```

Comment:
This is waaaaay too complex. Use the shortest selector that would uniquely identify the target element.